src/nri/sampler.go

```diff
@@ -274,16 +275,16 @@ func blkio(bio *biz.BlkIO) []entry {
 	var entries []entry
 
 	if bio.TotalReadCount != nil {
-		entries = append(entries, metricIOTotalReadCount(bio.TotalReadCount), metricIOReadCountPerSecond(bio.TotalReadCount))
+		entries = append(entries, metricIOTotalReadCount(*bio.TotalReadCount), metricIOReadCountPerSecond(*bio.TotalReadCount))
```

Comment:
Good catch! We're testing the biz metrics in the `TestBlkIOMetrics` but not the sampler 👍